src/app/ReadHandler.h

```diff
@@ -280,7 +280,13 @@ class ReadHandler : public Messaging::ExchangeDelegate
     // are waiting for the max reporting interval to elaps.  When mHoldSync
     // becomes false, we are allowed to send an empty report to keep the
     // subscription alive on the client.
-    bool mHoldSync                   = false;
+    bool mHoldSync = false;
+    // For subscriptions, we record the timestamp when we started to generate the last report.
+    // The mCurrentReportTick records the timestamp for the current report, which won;t be used for checking if this
+    // ReadHandler is dirty.
+    // mLastReportTick will be set to mCurrentReportTick after we sent the last chunk of the current report.
+    uint64_t mLastReportTick         = 0;
+    uint64_t mCurrentReportTick      = 0;
```

Comment:
This is more aptly the beginning time of the current series of reports, i.e `mCurrentReportsBeginTime`.